packages/irmin/irmin.0.11.0/opam

```diff
@@ -47,7 +47,7 @@ depends: [
 depopts: ["base-unix" "git" "git-unix" "cohttp" "mirage-git"]
 conflicts: [
   "cohttp" {< "0.18.3"}
-  "git" {< "1.8.0"}
+  "git" {< "1.8.0" & >= "1.10.0"}
```

Comment:
Is `&` still the right way to write conflicts against version ranges?  It seems like a mistake, since `conflicts` should be a disjunction, i.e. 

```
 conflicts: [
  "git" {< "1.8.0"}
  "git" {>= "1.10.0"}
```
should be equivalent to

```
 conflicts: [
  "git" {< "1.8.0" | >= "1.10.0"}
```

(related: https://github.com/ocaml/opam/issues/2267)